microsoft/testsuites/xfstests/xfstests.py

```diff
@@ -9,447 +9,448 @@
 from lisa.executable import Tool
 from lisa.messages import SubTestMessage, TestStatus, create_test_result_message
 from lisa.operating_system import (
-    CBLMariner,
-    Debian,
-    Oracle,
```

Comment:
Can you clarify why changes were intended in this file?

Reply:
I am just copying a recent change by Lili. This is not my file.